src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -778,6 +779,37 @@ public void scrollToPosition(String selector) {
     jsActions.scrollToSpecificElement(driver.findElement(By.cssSelector(selector)));
     PageObjectLogging.log("scrollToSelector", "Scroll to the web selector " + selector, true);
   }
+  // This scroll has been created becouse of ADEN-4359
+  public void scrollToBottomLeaderboard(){
+    scrollToFooter();
+    scrollTo(presentLeaderboard);
+    scrollToFooter();
+  }
+
+  public void clickOnAdImage(String adSlotName){
+    driver.findElement(By.cssSelector(String.format(AD_SLOT_SELECTOR, adSlotName))).click();
+    PageObjectLogging.log("clickOnAdImage", adSlotName + " is clicked", true);
+  }
+
+  public void switchToSecondTab(String firstTab) {
+    ArrayList<String> openTabs = new ArrayList(driver.getWindowHandles());
+    for (String window : openTabs) {
+      if (window != firstTab) {
+        driver.switchTo().window(window);
+      }
+    }
+  }
+
+  public boolean isAdDisplayed(String adSlotName) {
+    try {
+      System.out.println(String.format(adSlotName, adSlotName));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS106)